setup.cfg

```
@@ -91,6 +91,7 @@ filterwarnings =
     ignore::DeprecationWarning:bqplot_image_gl
     ignore::DeprecationWarning:bqscales
     ignore::DeprecationWarning:traittypes
+    ignore::Warning:specutils.spectra.spectrum1d
```

Everything subclasses `Warning` (except deprecation, I think), so this is a little redundant.
```suggestion
    ignore:::specutils.spectra.spectrum1d
```